packages/hd-transport-web-device/src/electron-ble-transport.ts

```
@@ -0,0 +1,365 @@
+      const cleanHexData = hexData.replace(/\s+/g, '');
+      if (!/^[0-9A-Fa-f]*$/.test(cleanHexData)) {
+        return { isComplete: false, error: 'Invalid hex data format' };
+      }
+
+      // Convert hex string to Uint8Array
+      const hexMatch = cleanHexData.match(/.{1,2}/g);
+      if (!hexMatch) {
+        return { isComplete: false, error: 'Failed to parse hex data' };
+      }
+
+      const data = new Uint8Array(hexMatch.map(byte => parseInt(byte, 16)));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)